assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -125,9 +127,14 @@ function BannerNotification( {
 	const persistDismissal = () =>
 		setItem( cacheKeyDismissed, new Date(), { ttl: null } );
 
+	const windowWidth = useWindowWidth();
 	const breakpoint = useBreakpoint();
 	const isMounted = useMountedState();
 
+	// There is a 1px difference between the tablet breakpoint determination in `useBreakpoint` and the `min-width: $bp-tablet` breakpoint the `@mixin googlesitekit-inner-padding` uses,
+	// which in turn is used by the notification. This why we are using `useWindowWidth` here, instead of the breakpoint returned by `useBreakpoint`.
+	const isMinWidthTablet = windowWidth >= 600;
```

Comment:
Should we instead change the breakpoint used by `@mixin googlesitekit-inner-padding`? While this may work, it seems like we might be fixing the problem in the wrong place.

Core includes 600px as the [max-width for mobile](https://github.com/WordPress/wordpress-develop/blob/1e420358957213691579b77e88aca23bd71265c9/src/wp-admin/css/common.css#L4069-L4070) rather than the beginning of tablet. This might be out of scope here too but maybe we need a more holistic revisiting of our breakpoints.

Reply:
I fully agree that it's not a nice fix and it would be preferable to address the breakpoint in the mixin.

The problem is, the mixin is used in a number of places (9 at present), and is included in some generic classes which are in heavy use. I know we have applied off-by-one-pixel fixes in a number of places to deal with discrepancies in breakpoint usage so I don't think it's a small task to change the breakpoint in the mixin, would be a significant chance of breaking things that might be hard to spot.

Furthermore the issue is not confined to the mixin and in fact points to a more fundamental 1px disconnect between our CSS vs our `useBreakpoint` and what's in core.

Taking tablet as an example, we define `$bp-tablet` as `600px`, and then use the media query `min-width: $bp-tablet` in a current count of 98 places. So we're using a lot of `min-width 600px` breakpoints.

https://github.com/google/site-kit-wp/blob/8eb43aa40dbf018ac330a9da6428953905cf744a/assets/sass/config/_variables.scss#L270

https://github.com/google/site-kit-wp/blob/8eb43aa40dbf018ac330a9da6428953905cf744a/assets/sass/config/_variables.scss#L279

![image](https://user-images.githubusercontent.com/18395600/201922306-7031630c-46c1-4368-83bc-7f6ca7a52580.png)

This contradicts the `max-width: 600px` for mobile you identified in core, as well as our `useBreakpoint` hook which also implements `max-width 600px` logic:

https://github.com/google/site-kit-wp/blob/8eb43aa40dbf018ac330a9da6428953905cf744a/assets/js/hooks/useBreakpoint.js#L47-L51

I think we need to do as you've identified in the latter part of your comment, take a more holistic look at our breakpoint usage.

This might be worthy of an Epic; maybe we could have an initial issue to investigate and determine how to proceed, what do you think? Let me know if you would like me to create an issue for this.

Incidentally, back to the topic of the line of code in question, I did make a suggestion in the provisional AC for [this issue](https://github.com/google/site-kit-wp/issues/6146) I created to potentially revert the fix that introduced this line, so it might go away in future anyway!

Reply:
Thanks for elaborating Tom, that sounds good to me. It would probably be good to wait until our VRT is working properly again as well.

We probably need an issue to audit the current state of things and then create smaller issues from there. I'll create a separate task for this to start with.